homework/index.js

```diff
@@ -30,18 +30,94 @@
     return elem;
   }
 
+  function render(parent, repositories) {
+    const table = createAndAppend('table', parent, {});
+    const tbody = createAndAppend('tbody', table, {});
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('td', tr, { class: 'label', text: 'Repository : ' });
+    const td = createAndAppend('td', tr);
+    createAndAppend('a', td, {
+      text: repositories.name,
+      href: 'https://github.com/HackYourFuture/alumni',
```

Comment:
Imagine the situation when the repositoy name changes, the link going to be broken, right?
To fix this, I think you already did it in line 69:
```suggestion
      href: repositry.html_url, // assuming you applied the change on the singuler naming form.
```